composer/loggers/in_memory_logger.py

```diff
@@ -74,3 +75,50 @@ def log_metric(self, timestamp: Timestamp, log_level: LogLevel, data: TLogData):
             self.data[k].append((timestamp, log_level, v))
         self.most_recent_values.update(data)
         self.most_recent_timestamps.update({k: timestamp for k in data})
+
+    def get_timeseries(self, metric: str, time_unit: Union[str, None] = None):
```

Comment:
For TimeUnit, we generally take the string or enum value:

```suggestion
    def get_timeseries(self, metric: str, time_unit: Optional[Union[str, TimeUnit]] = None):
```

Reply:
TimeUnits are a little clunky here because TimeStamp fields don't follow the same convention: they're named `epoch`, `batch`, `batch_in_epoch`, etc.